dash_bio/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dash-bio",
-  "version": "0.0.8rc3",
+  "version": "0.0.8-rc5",
```

Comment:
Are we changing this notation based on a convention? If so, which one?

Reply:
We'll be using the dash going forward. I don't think it's particularly important.

Reply:
(Just to be clear, the TestPyPI package removes the dash from the version name regardless!)

Reply:
I don't think either, I just like to have a reference (link) to other instances or discussions... That's how you build an ecosystem.

Reply:
(Yes, this I gathered, that's why I wanted to see where the inspiration came from, for adding the dash in the version number we display.)

Reply:
There I go with a reference (convention for hyphenating release candidate version notation): https://semver.org/spec/v2.0.0-rc.2.html